Emotion - fix integration tests (#8187)
bbc/simorgh: src/app/components/Grid/README.md

## Usage

> When using one of the extended Grid components, e.g. `FrontPageGrid`, if you wish to extend it using the `as` prop, following the Emotion migration you can now just pass the `as` prop to that component: emotion does not support `forwardedAs`.

Reviewer: ```suggestion
When using an extended Grid component, e.g. `FrontPageGrid`, you can change the element that's rendered by using the `as` prop.
```

e.g.

> `<Grid as="main" role="main ....>...</Grid>`
> `<FrontPageGrid as="main" role="main ....>...</FrontPageGrid>`

Reviewer: ```suggestion
`<Grid as="main" role="main>...</Grid>`
`<FrontPageGrid as="main" role="main>...</FrontPageGrid>`
```
